docs/likelihood.rst

```diff
@@ -199,11 +199,13 @@ each sample would yield a very large number of nuisance parameters with limited
 utility. Therefore a set of bin-wise scale factors :math:`\gamma_b` is
 introduced to model the overall uncertainty in the bin due to MC statistics.
 The constraint term is constructed as a set of Gaussian constraints with a
-central value equal to unity for each bin in the channel. The scales
-:math:`\sigma_b` of the constraint are computed from the individual
+central value equal to unity (one Gaussian for each bin). The scales
```

Comment:
Well, one can _change_ the constraint for the staterror typically... so I'm slightly concerned about specifically mentioning the distribution here.

Reply:
what's a possible alternative? I can add `e.g.` maybe?

Reply:
Well, I think we mention Gaussian already originally so maybe... leave this for now.